### ISSUE TITLE
Ｅｒｒｏｒｓ ｗｈｅｎ ｏｕｔｐｕｔ ｉｓ ｎｕｍｐｙｄｏｃ

### ISSUE THREAD

**Issue by masasin** [dadadel/pyment#12]
Ｆｏｒ ｍａｎｙ ｆｉｌｅｓ， Ｉ ｇｅｔ ｔｈｅ ｆｏｌｌｏｗｉｎｇ ｅｒｒｏｒ：

```ｐｙｔｈｏｎ
Traceback (most recent call last):
  File "/usr/local/bin/pyment", line 4, in <module>
    __import__('pkg_resources').run_script('Pyment==0.3.0', 'pyment')
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 534, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 1445, in run_script
    exec(script_code, namespace, namespace)
  File "/usr/local/lib/python2.7/dist-packages/Pyment-0.3.0-py2.7.egg/EGG-INFO/scripts/pyment", line 145, in <module>
    
  File "/usr/local/lib/python2.7/dist-packages/Pyment-0.3.0-py2.7.egg/EGG-INFO/scripts/pyment", line 142, in main
    
  File "/usr/local/lib/python2.7/dist-packages/Pyment-0.3.0-py2.7.egg/EGG-INFO/scripts/pyment", line 97, in run
    
  File "build/bdist.linux-x86_64/egg/pyment/pyment.py", line 286, in proceed
  File "build/bdist.linux-x86_64/egg/pyment/docstring.py", line 1812, in generate_docs
  File "build/bdist.linux-x86_64/egg/pyment/docstring.py", line 1789, in _set_raw
  File "build/bdist.linux-x86_64/egg/pyment/docstring.py", line 1709, in _set_raw_return
TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'
```

Ｔｈｅ ｃｏｍｍａｎｄ ｗａｓ `ｐｙｍｅｎｔ －ｉ "google" －ｏ ＂ｎｕｍｐｙｄｏｃ＂ ｍｙｆｉｌｅ．ｐｙ`.

**Comment by dadadel** [comment]
Hi @masasin!

Thanks for reporting issue.
Can you please provide an basic example of docstrings implying that error ?

**Comment by masasin** [comment]
This has been fixed.

**Comment by dadadel** [comment]
I close it as it is fixed in v0.3.1
